www/src/css/layout.css

```diff
@@ -37,3 +35,8 @@
   width: 100%;
   z-index: 10000;
 }
+
+.grey-wrapper {
```

Comment:
Any reason why it doesn't have `.b-` prefix (for new BEM styles)?
Is it an old class that is reused or a new one?

I also noticed that it introduces a new wrapper div in HTML. Can it be used as a second class on existing `inner-wrapper` div?